.github/workflows/unit_and_integration_tests.yaml

```
@@ -12,6 +18,8 @@
           ruby-version: 3.1.4
           # runs 'bundle install' and caches installed gems automatically
           bundler-cache: true
+      - name: Environment Variables
+        run: echo $CONFLUENCE_URL
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:6: Double quote to prevent globbing and word splitting

(shellcheck)